salt/modules/opkg.py

```diff
@@ -524,8 +524,17 @@ def install(name=None,
                 else:
                     to_install.append(pkgname)
             else:
-                pkgstr = '{0}={1}'.format(pkgname, version_num)
                 cver = old.get(pkgname, '')
+                version_conditions = [x.strip() for x in version_num.split(',')]
+                for version_condition in version_conditions:
+                    (version_string, version_operator, operator_specified) = _get_version_info(version_condition)
+                    if operator_specified:
+                        # Version conditions are sent to the solver as install commands
+                        pkgstr = '{0}{1}{2}'.format(pkgname, version_operator, version_string)
+                        to_install.append(pkgstr)
+                        cmd_prefix.append('--force-downgrade')
+                    else:
+                pkgstr = '{0}={1}'.format(pkgname, version_num)
```

Comment:
formatting looks wrong here and python is very sensitive to that so I assume python is going to fail loading this file.

Reply:
Thank you! I didn't observe when I copied the code here. I corrected it.